packages/flutter_driver/lib/src/driver.dart

```diff
@@ -251,6 +266,14 @@ class FlutterDriver {
     return response['response'];
   }
 
+  void _logCommunication(String message)  {
+    if (_printCommunication)
+      _log.info(message);
+    f.File file = fs.file(p.join(testOutputsDirectory, 'flutter_driver_commands.log'));
```

Comment:
Also this will result in failure if run against a read-only file system or a file system that has no more free disk space.

Maybe we should make saving to disk also optional?